source/ore/publish.rst

```diff
@@ -145,3 +145,57 @@ Support
 -------
 
 Support cannot do anything, merely a way of showing that the member is a part of the project.
+
+
+Documenting Your Project With Pages
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To document your plugin Ore offers the creation of pages, by default you get a 'Home' page when you create your new Project.
+To add new pages you can click on the '+' icon in the Pages bar on the right of the screen, doing so will open a pop-up where you 
+can choose a name and the parent page (adding a page on the Home page is not possible). If you choose the '<None>' option the page 
+will become a root page where you can later add child pages on, if you hover choose to add it to an other page the page will only show
```

Comment:
Again, please replace comma with full stop and begin a new sentence. "If you hover  ..."
Secondly, I'm not sure exactly what "If you hover choose to add it" means ...  can you rephrase that so it's clear?
Also, please replace "an other" -> "another"
Add comma, lose one "page"  - i.e. "... to another page, it will only appear when the parent page is expanded"